Autopilot/AttitudeManager/net.h

```diff
@@ -15,16 +15,16 @@ extern "C" {
 #endif
 
 /** Time in miliseconds for how often a P0(high priority) packet gets sent down. Default=300 **/
-#define P0_SEND_FREQUENCY 300 
+#define P0_SEND_FREQUENCY 200 
```

Comment:
Since this isn't a proper operating operating system, this might cause problems. Be careful for future datalink problems (especially with uplink)